src/lib/jdk/KeyTool.ts

```diff
@@ -66,10 +66,10 @@ export class KeyTool {
       '-genkeypair',
       `-dname "cn=${keyOptions.fullName}, ou=${keyOptions.organizationalUnit}, ` +
           `o=${keyOptions.organization}, c=${keyOptions.country}"`,
-      `-alias ${keyOptions.alias}`,
-      `-keypass ${keyOptions.keypassword}`,
-      `-keystore ${keyOptions.path}`,
-      `-storepass ${keyOptions.password}`,
+      `-alias \"${keyOptions.alias}\"`,
```

Comment:
You don't need the backslashes here since we're using a template string (eg, there's just `"` instead of `\"` on line 67).

Reply:
PS: Even though it's not needed, the output is correct with the escapes.